### ISSUE TITLE
Expand button falls off its black background when the adjacent title wraps to 2 lines at 200% zoom (Chrome Zoom Text Only)

### ISSUE THREAD

**Issue by dotherightthing** [dotherightthing/wpdtrt-gallery#108]
* WCAG - 1.4.4 Resize text (Level AA, WCAG 2.0)
* dont-believe-the-hype-evaluation-report.html
